fe/src/main/java/org/apache/doris/catalog/EsTable.java

```diff
@@ -294,6 +331,13 @@ public void readFields(DataInput in) throws IOException {
             } else {
                 enableKeywordSniff = true;
             }
+            if (tableContext.containsKey("maxDocValueFields")) {
+                try {
+                    maxDocValueFields = Integer.parseInt(tableContext.get("maxDocValueFields"));
+                } catch (Exception e) {
+                    maxDocValueFields = 30;
+                }
+            }
```

Comment:
if the tableContext don't contain "maxDocValueFields", the maxDocValueFields should set Default_max_docvaues_fields or set MaxInt.